.github/ISSUE_TEMPLATE/custom_product/tech-day.md

```diff
@@ -49,46 +65,56 @@ This checklist is focused on a classic Jahia repository (module, app)
 - [ ] 🟠 Dependencies listed in packages.json are no more than 2 major versions behind their latest release
 - [ ] 🟠 Linting is executed properly and show no warnings
 - [ ] 🟠 No warning are presents in the browser console when using the app
+- [ ] 🙏 Remove unreachable code
 ### Java
 - [ ] 🔝 Java dependencies are explicitly declared in the module's pom.xml
 - [ ] 🔝 Spring is not used in the module
-- [ ] 🔝 No warnings or errors are present when building the module locally or on GitHub Actions
-- [ ] 🟠 No code smell on [Sonarqube](https://sonarqube.jahia.com/projects) for the module
 ### Security
-- [ ] 🔝 Our security lead confirmed there are no known security vulnerabilities affecting this codebase
-### QA
+- [ ] 🚨 SBOM is generated (configuration available [here](https://jahia-confluence.atlassian.net/wiki/spaces/PR/pages/2071358/3rd-party+libraries+-+Ref+ISPOL08.A14024#%5BinlineExtension%5DSBOM-creation-and-OWASP-Dependency-Track)) and uploaded to [Dependency Track](https://dependency-track-prod.jahia.com/)
+- [ ] 🔝 I've reviewed the security [vulnerabilities](https://sonarqube.jahia.com/issues?resolved=false&types=VULNERABILITY) and [hotspots](https://sonarqube.jahia.com/security_hotspots?id=org.jahia.server%3Ajahia-root) affecting this codebase and discussed it with the Security lead before taking action (Create a [SECURITY](https://support.jahia.com/browse/SECURITY) ticket, Close as false-positive, etc.)
+- [ ] 🔝 I've reviewed the [vulnerabilities](https://dependency-track-prod.jahia.com/login?redirect=%2Fprojects) affecting the libraries used by the module (related [documentation](https://jahia-confluence.atlassian.net/wiki/spaces/PR/pages/2079156/Analyzing+vulnerabilities+in+3rd+party+libraries)) and discussed it with the Security lead before taking action (Create a [SECURITY](https://support.jahia.com/browse/SECURITY) ticket, Close as false-positive, etc.)
+- [ ] 🔝 A job running Sonar checks (including OWASP Dependency Check) is executed regularly
+### QA / Automated Tests
+- [ ] 🚨 The codebase is compatible with the latest release of Jahia
 - [ ] 🔝 Automated tests are using jahia-cypress for all utils functions
```

Comment:
This might not true if the utility is meant to be use by this module only. We have to share the most we can, but not always.